src/test/java/com/uid2/shared/vertx/VertxUtilsTest.java

```diff
@@ -0,0 +1,30 @@
+package com.uid2.shared.vertx;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class VertxUtilsTest {
+    @Test
+    void parseClientAppVersionSuccess() {
+        // example input uid2-operator=2.7.16-SNAPSHOT;uid2-attestation-api=1.1.0;uid2-shared=2.7.0-3e279acefa
+        final String input = "uid2-operator=2.7.16-SNAPSHOT;uid2-attestation-api=1.1.0;uid2-shared=2.7.0-3e279acefa";

```

Comment:
Is it always going to be in the same order?

Reply:
Yes, it is set by the Core client, so we own the production of this. It is for reporting purposes